server/sonar-db-dao/src/main/resources/org/sonar/db/organization/OrganizationMapper.xml

```diff
@@ -7,6 +7,7 @@
     org.kee as "key",
     org.name as "name",
     org.description as "description",
+    org.default_quality_gate_uuid as "defaultQualityGateUuid",
```

Comment:
This should not be needed, using QualityGateDao#selectDefault should be enough to get the default quality gate of an organization